Makefile

```diff
@@ -69,6 +69,9 @@ license-check:
 test:
 	go test -cover ./...
 
+test-docker:	
```

Comment:
can you add a description for top-level make targets? Why would someone run this?